remoulade/api/main.py

```diff
@@ -0,0 +1,35 @@
+""" This file describe the API to get the state of messages """
+from flask import Flask, request
+from werkzeug.exceptions import BadRequest, HTTPException, NotFound
```

Comment:
could you add `"remoulade.api"` to the package line in setup.py line 65 ? It make it possible to import the given sup package.

And add app to remoulade/api/__init__.py, don't forget to add it to __all__ (look at other inits).

Then could you write le little changelog ? 

Thanks !